drafts/draft-ietf-ippm-ioam-data.xml

```diff
@@ -1225,9 +1227,9 @@ The trace option data MUST be 4-octet aligned:
 
               <t hangText="buffer occupancy:">4-octet unsigned integer field.
               This field indicates the current status of the buffer occupancy.
-              The buffer occupancy is expressed as the current number of
-              memory buffers used by the set of queues that share a common
-              buffer pool. <figure>
```

Comment:
I think the edits are going a little too far, cutting out much of the definition of buffer occupancy. The text "used by the set of queues that share a common buffer pool" should be reinserted somehow. How about:

This field indicates the current status of the occupancy _of the common buffer pool used by a set of queues_. The units of this field depend on the equipment type and deployment and has to be interpreted within the context of a namespace and/or node-id if used.

Reply:
I also wonder about the last sentence. Isn't that true for almost every field we have. Do we really need to state it explicitly?

Reply:
"This field indicates the current status of the occupancy of the common buffer pool used by a set of queues. The units of this field depend on the equipment type and deployment and has to be interpreted within the context of a namespace and/or node-id if used." looks good. 

I would prefer to keep the explicit mention of the units defined by equipment type that can be derived by the node-id along with the namespace reference.

Reply:
The change in the first paragraph just above, relating to my first comment above, has not been incorporated.

Reply:
> The change in the first paragraph just above, relating to my first comment above, has not been incorporated.

But we are already stating it at the beginning of the section 4.2.2 - "All the data fields MUST be 4-octet aligned." Should we re-state it or move it around?

Reply:
Not referring to that. Currently it reads:

> This field indicates the current status of the buffer occupancy.

The term buffer occupancy is not defined anywhere, though it used to say something about a buffer pool shared by a set of queues.

My suggestion is:

> This field indicates the current status of the occupancy of the common buffer pool used by a set of queues.

Reply:
@mickeyspiegel https://github.com/inband-oam/ietf/commit/f203b842e73a60e4fdc133dc7ec55da8e7551325 was pushed to address this.